src/components/EmptyState/README.md

```diff
@@ -157,6 +157,11 @@ Use to explain a single feature before merchants have used it.
   heading="Manage your inventory transfers"
   action={{content: 'Add transfer'}}
   secondaryAction={{content: 'Learn more', url: 'https://help.shopify.com'}}
+  footer={
+    <p>
+      Import inventory from <a href="/settings">settings</a>
```

Comment:
use `<Link>` instead of `<a>` so that the link is colored correctly in the demo (these examples eventually get shown as part of our styleguide and in storybook)

At https://percy.io/Shopify/polaris-react/builds/1579481/view/104454203/1280?browser=chrome&mode=head you can see the link is the default blue rather than the polaris link blue.